applications/nrf5340_audio/src/utils/nrf5340_audio_dk.c

```diff
@@ -35,7 +35,7 @@ static int hfclock_config_and_start(void)
 	return 0;
 }
 
-static int leds_set(void)
+static int leds_set(bool led_override, enum led_color color)
```

Comment:
Suggest a more descriptive name. Or rather, this can be a vararg, and if color is submitted it will set this custom color.

Reply:
Change function name to center_led_set